File: main/src/main/java/net/fwitz/math/main/binary/split/functions/hyperbolic/SplitCoshZ.java
package net.fwitz.math.main.binary.split.functions.hyperbolic;

import net.fwitz.math.binary.split.SplitComplex;
import net.fwitz.math.plot.binary.split.SplitComplexFunctionPlot;

public class SplitCoshZ {
    public static void main(String[] args) {
        new SplitComplexFunctionPlot("cosh(z)")
                .computeFn(SplitComplex::cosh)
                .render();
    }
}

File: main/src/main/kotlin/net/fwitz/math/main/binary/split/functions/hyperbolic/SplitCoshZ.kt
package net.fwitz.math.main.binary.split.functions.hyperbolic

import net.fwitz.math.binary.split.SplitComplex
import net.fwitz.math.plot.binary.split.SplitComplexFunctionPlot

object SplitCoshZ {
    @JvmStatic
    fun main(args: Array<String>) = SplitComplexFunctionPlot("cosh(z)")
        .computeFn(SplitComplex::cosh)
        .render()
}